client/my-sites/plans-grid/components/comparison-grid/index.tsx

```diff
@@ -473,7 +475,8 @@ const ComparisonGridHeaderCell = ( {
 				isLaunchPage={ isLaunchPage }
 				planSlug={ planSlug }
 				flowName={ flowName }
-				onUpgradeClick={ () => onUpgradeClick( planSlug ) }
+				freeTrialPlanSlug={ freeTrialPlanSlug }
+				onUpgradeClick={ ( overridePlanSlug ) => onUpgradeClick( overridePlanSlug ?? planSlug ) }
```

Comment:
Similar here. We can wrap this with `useCallback` to pass same reference on rerenders. I know it wasn't done before, so totally fine to ignore.

Reply:
I can't really see how it'd be useful. The function will be redefined anyway and its cost is marginal. Rendering will happen either way as soon as something else in the tree changes, so I don't think we should bother that much about it.